rust/stub/src/main.rs

```diff
@@ -238,12 +243,27 @@ fn main(handle: Handle, mut system_table: SystemTable<Boot>) -> Status {
         warn!("Hash mismatch for initrd!");
     }
 
+    if tpm_available(system_table.boot_services()) {
+        info!("TPM available, will proceed to measurements.");
+    }
```

Comment:
Shouldn't we just not measure anything if there is no TPM available? i.e. only call `measure_image` in this `if` clause.

Reply:
Good point!